executor/src/main/java/com/redhat/service/bridge/executor/ExecutorsService.java

```diff
@@ -31,25 +27,17 @@
     @Inject
     ExecutorsProvider executorsProvider;
 
-    public void init(@Observes StartupEvent ev) {
-        ExtensionProvider.getInstance().registerExtension(BridgeCloudEventExtension.class, BridgeCloudEventExtension::new);
-    }
-
     @Incoming(EVENTS_IN_TOPIC)
```

Comment:
Does it mean that the annotation parameter doesn't have any effect any more, as the topic is set using properties?

Reply:
@sutaakar the `bridgeId` cloud event extension was a leftover from the first implementation of the architecture that was using one single `events` topic. The executor was then checking that the `bridgeId` was equal to the `processor.getBridgeId()` to ensure it had to consume the message. 

Now that the ingress is sending events to its dedicated topic and the processors are consuming events from such topic, the bridgeId extension is not needed anymore. 

Does this answer your question?

Reply:
@r00ta yes, I get it. I am just referring to the `EVENTS_IN_TOPIC` parameter of `@Incoming` annotation. What is the parameter purpose now when the topic is defined using properties?

Reply:
`EVENTS_IN_TOPIC` is the Reactive Stream channel name: you need first to "bind" the incoming annotation with a channel and then configure the channel using the [properties](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/executor/src/main/resources/application.properties#L16-L21)

Reply:
I see now.
Could you please adjust the javadoc for `EVENTS_IN_TOPIC`, possibly rename the constant to something like `EVENTS_IN_CHANNEL`, to avoid possible future confusions?

Reply:
Makes sense, I'll rename it in this PR